### PR TITLE
fixed the keyboard accessibiilty bug in area chart on legend click

### PR DESCRIPTION
**Pull request by yush-singla**
## Previous Behavior

- Keyboard focus is not landing on the particular legend on graph when selecting the particular legend Option
- The color of the vertical line in line chart didn't have the correct luminosity ratio (3:1)

## New Behavior

- Keyboard focus is landing on the particular legend on graph when selecting the particular legend Option
- The color of the vertical line in the line chart has the correct luminosity ratio (3:1)
image

## Related Issue(s)

- Fixes # https://uifabric.visualstudio.com/iss/_workitems/edit/8692
- Fixes # https://uifabric.visualstudio.com/iss/_workitems/edit/8471

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/57de08c18915440cdc54a1242bbdf948e2b0f562'>57de08c18915440cdc54a1242bbdf948e2b0f562</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/components/AreaChart/AreaChart.base.tsx [hunk 1]
```diff
@@ -695,7 +699,7 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
               onMouseOut={this._onRectMouseOut}
               onMouseOver={this._onRectMouseMove}
               {...(this.props.optimizeLargeData && {
-                'data-is-focusable': true,
+                'data-is-focusable': this.state.selectedLegendIndex === index || this.state.selectedLegendIndex === -1,
```
[the remark this answers is not in the document]
Comment (yush-singla) [reply packages/react-charting/src/components/AreaChart/AreaChart.base.tsx]: Thanks for pointing it out, Resolved

File: packages/react-charting/src/components/AreaChart/AreaChart.base.tsx [hunk 2]
```diff
@@ -695,7 +695,7 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
               onMouseOut={this._onRectMouseOut}
               onMouseOver={this._onRectMouseMove}
               {...(this.props.optimizeLargeData && {
-                'data-is-focusable': true,
+                'data-is-focusable': points[index].legend === this.state.selectedLegend,
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/AreaChart/AreaChart.base.tsx:698]: use points[index]!.legend

Comment (AtishayMsft) [reply packages/react-charting/src/components/AreaChart/AreaChart.base.tsx:698]: I see this comment is still not resolved.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**